test/management/branding.tests.js

```diff
@@ -233,4 +233,244 @@ describe('BrandingManager', function() {
       });
     });
   });
+
+  describe('#getUniversalLoginTemplate', function() {
+    beforeEach(function() {
+      this.request = nock(API_URL)
+        .get('/branding/templates/universal-login')
+        .reply(200);
+    });
+
+    afterEach(function() {
+      nock.cleanAll();
+    });
+
+    it('should accept a callback', function(done) {
+      this.branding.getUniversalLoginTemplate(function() {
+        done();
+      });
+    });
+
+    it('should return a promise if no callback is given', function(done) {
+      this.branding
+        .getUniversalLoginTemplate()
+        .then(done.bind(null, null))
+        .catch(done.bind(null, null));
+    });
+
+    it('should pass any errors to the promise catch handler', function(done) {
+      nock.cleanAll();
+
+      var request = nock(API_URL)
+        .get('/branding/templates/universal-login')
+        .reply(500);
+
+      this.branding.getUniversalLoginTemplate().catch(function(err) {
+        expect(err).to.exist;
```

Comment:
I just copied this from other tests in the same file, but note that by not returning the promise to mocha and instead relying on `done`, if any test fails within a promise, the error reported by mocha will just be a timeout.